app/v1/connectors/DeletePensionsReliefsConnector.scala

```diff
@@ -41,7 +41,7 @@ class DeletePensionsReliefsConnector @Inject() (val http: HttpClient, val appCon
       if (taxYear.useTaxYearSpecificApi) {
         TaxYearSpecificIfsUri[Unit](s"income-tax/reliefs/pensions/${taxYear.asTysDownstream}/$nino")
       } else {
-        DesUri[Unit](s"income-tax/reliefs/pensions/$nino/${taxYear.asDownstream}")
+        DesUri[Unit](s"income-tax/reliefs/pensions/$nino/${taxYear.asMtd}")
```

Comment:
Is it worth commenting that this is deliberate? As obviously this isn't MTD so it looks incorrect.